app/models/api/task.rb

```diff
@@ -20,5 +20,11 @@ def late?
     def reporting_period
       [Date::MONTHNAMES[period_month], period_year].join(' ')
     end
+
+    def completed_at
+      return unless latest_submission.submitted_at
+
+      Time.zone.parse(latest_submission.submitted_at).to_s(:date_with_utc_time)
```

Comment:
Do we really want to be showing suppliers users timestamps in the format "15 February 2019 09:59 UTC"? That doesn't feel particularly user-friendly. Could we show it to them in the GB timezone instead?

Reply:
We did talk about this, the decision was made to show the time as it is stored (UTC) until we make a call application wide how to handle time zones.